RecoHGCal/TICL/plugins/TrackstersMergeProducer.cc

```diff
@@ -285,7 +285,8 @@ void TrackstersMergeProducer::produce(edm::Event &evt, const edm::EventSetup &es
   }
 
   // Neutral Hadrons
-  constexpr float mpion2 = 0.13957f * 0.13957f;
+  constexpr double mpion = 0.13957f;
```

Comment:
```suggestion
  constexpr double mpion = 0.13957;
```
this is a minor discrepancy wrt the master branch; I'm not sure it's a desired source of numerical differences in results between releases.